src/f3dasm/_src/experimentdata/experimentdata.py

```diff
@@ -986,15 +1039,18 @@ def domain_factory(domain: Domain | None, input_data: _Data) -> Domain:
         raise TypeError(f"Domain must be of type Domain or None, not {type(domain)}")
 
 
-def jobs_factory(jobs: Path | str | _JobQueue | None, input_data: _Data, job_value: Status) -> _JobQueue:
+def jobs_factory(jobs: Path | str | _JobQueue | None, input_data: _Data,
+                 output_data: _Data, job_value: Status) -> _JobQueue:
```

Comment:
A method that handles private arguments and returns private object should be private as well.

Returns is missing on method dosctring.

Reply:
Upon expecting I see that the return on the docstring is present?

Reply:
I have added the request for private methods in a new issue #206